helios-services/src/main/java/com/spotify/helios/servicescommon/statistics/NoopZooKeeperMetrics.java

```diff
@@ -17,7 +17,14 @@
 
 package com.spotify.helios.servicescommon.statistics;
 
+import com.codahale.metrics.Timer;
+
 public class NoopZooKeeperMetrics implements ZooKeeperMetrics {
   @Override
   public void zookeeperTransientError() {}
+
+  @Override
+  public Timer timer(String name) {
+    return new Timer();
```

Comment:
We shouldn't use a real timer, that actually gathers and keeps data, for a "no op" implementation. Unfortunately `Timer` isn't an interface so we can't really return a no-op timer...